### ISSUE TITLE
Sonata

### ISSUE THREAD

**Issue by Leinnan** [snwh/paper-icon-theme#203]
### Icon Request

- Sonata
- GTK mpd client
- Icon=sonata

The player has of course panel icons stored in /usr/share/pixmaps. 

- sonata.png
![sonata](https://cloud.githubusercontent.com/assets/13188195/15456006/ab8fbb34-2063-11e6-8e76-d654d7a822d5.png)

- sonata_disconnect.png

![sonata_disconnect](https://cloud.githubusercontent.com/assets/13188195/15456002/9857e032-2063-11e6-93ff-bf7cfbde37f6.png)
- sonata_pause.png

![sonata_pause](https://cloud.githubusercontent.com/assets/13188195/15456003/9cb16a2c-2063-11e6-9331-46a4a4defc45.png)
- sonata_play.png

![sonata_play](https://cloud.githubusercontent.com/assets/13188195/15456005/a1d078e0-2063-11e6-9c13-1b69eeb2f2f9.png)

**Comment by snwh** [comment]
Why does this app have it's own play/pause icons?

**Comment by Leinnan** [comment]
Yes, all of them look like pic rel on XFCE with Paper Icons/Theme.
![2016-05-23-154710_1920x1080_scrot](https://cloud.githubusercontent.com/assets/13188195/15472930/cac737b2-20fd-11e6-87b7-da85cb0727b4.png)
